src/dotenv-vault.ts

```diff
@@ -0,0 +1,116 @@
+// Common Options
+const yesOption: Fig.Option = {
+  name: ["--yes", "-y"],
+  description:
+    "Automatic yes to prompts. Assume yes to all prompts and run non-interactively",
+};
+
+const helpOption: Fig.Option = {
+  name: "--help",
+  description: "Get documentation for the command",
+};
+
+const dotenvMeOption: Fig.Option = {
+  name: ["-m", "--dotenvMe"],
+  description:
+    "Pass .env.me (DOTENV_ME) credential directly (rather than reading from .env.me file)",
+  args: {
+    name: "DOTENV_ME",
+    description: "Pass .env.me (DOTENV_ME) credential directly",
+  },
+};
+
+// Common Arguments
+const dotenvVaultArg = {
+  name: "DOTENV_VAULT",
+  description: "Set .env.vault identifier. Defaults to generated value",
+  isOptional: true,
+};
+
+const dotenvMeArg = {
+  name: "DOTENV_ME",
+  description: "Set .env.me credential. Defaults to generated value",
+  isOptional: true,
+};
+
+const environmentArg = {
+  name: "ENVIRONMENT",
+  description: "Set environment to open to. Defaults to development",
+  isOptional: true,
+};
+
+const filenameArg = {
+  name: "FILENAME",
+  description:
+    "Set input filename. Defaults to .env for development and .env.{environment} for other environments",
+  isOptional: true,
+};
```

Comment:
Adding `template: "filepaths"` to an arg gives you `cd`-like suggestions!

```suggestion
const filenameArg = {
  name: "FILENAME",
  description:
    "Set input filename. Defaults to .env for development and .env.{environment} for other environments",
  isOptional: true,
  template: "filepaths",
};
```

Reply:
This is really great!

Any plans to let me filter the type of files in the filepath? Ideally I would only want to display `.env*` files.

So a glob pattern here would be really wonderful. Something like `template: "filepaths:.env*"`

Reply:
Ahhh yes this is possible! We have `@fig/autocomplete-generators`, you can use a regex.

```ts
import { filepaths } from "@fig/autocomplete-generators";
```

Pretty sure this regex should work for you:
```ts
const filenameArg = {
  name: "FILENAME",
  description:
    "Set input filename. Defaults to .env for development and .env.{environment} for other environments",
  isOptional: true,
  generators: filepaths({ matches: /^\.env\..+$/ }),
};
```

Example from bun:
https://github.com/withfig/autocomplete/blob/ed9580a7e411f9a679a537eaa419e84d8fb7f09c/src/bun.ts#L100